clang.nuspec.template

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>clang</id>
+    <version>$($tvVersion)</version>
+    <packageSourceUrl>https://github.com/bimmlerd/clang-choco</packageSourceUrl>
+    <owners>bimmlerd, petemounce</owners>
+
+    <title>LLVM compiler infrastructure clang (Install)</title>
+    <authors>LLVM team</authors>
+    <projectUrl>http://clang.llvm.org/</projectUrl>
+    <!--<iconUrl>http://cdn.rawgit.com/__REPLACE_YOUR_REPO__/master/icons/clang.png</iconUrl>-->
```

Comment:
I don't have an icon URL at all, and it's commented as a reminder to get one. Was planning to ask the LLVM team for a PR to fix it if they care to.